packages/mainsail/source/networks/mainsail.mainnet.ts

```diff
@@ -22,27 +22,46 @@ const network: Networks.NetworkManifest = {
 	},
 	hosts: [
 		{
-			host: "https://ark-live.arkvault.io/api",
+			host: "https://dwallets-evm.mainsailhq.com/api",
 			type: "full",
 		},
+		{
+			host: "https://dwallets-evm.mainsailhq.com/tx/api",
+			type: "tx",
+		},
 		{
 			host: "https://musig-demo.mainsailhq.com",
 			type: "musig",
 		},
 		{
-			host: "https://live.arkscan.io",
+			host: "https://explorer-evm-test.mainsailhq.com",
 			type: "explorer",
 		},
+		{
+			host: "https://dwallets-evm.mainsailhq.com/evm/api",
+			type: "evm",
+		},
 	],
 	id: "mainsail.mainnet",
 	importMethods,
-	knownWallets: "https://raw.githubusercontent.com/ArkEcosystem/common/master/mainnet/known-wallets-extended.json",
+	knownWallets:
+		"https://raw.githubusercontent.com/ArkEcosystem/common/master/mainsail/devnet/known-wallets-extended.json",
 	meta: {
-		fastDelegateSync: true,
-		nethash: "6e84d08bd299ed97c212c886c98a57e36545c8f5d645ca7eeae63a8bd62d8988",
+		chainId: 10_000,
+		// fastDelegateSync: true,
+		nethash: "a7206fa9d82a0896ddb26839e36eb75d4bb6da64a5e85eaf5d46a5435bef1fa5",
+		slip44: 111,
+		wif: 186,
 	},
 	name: "Mainnet",
-	transactions,
+	transactions: {
+		...transactions,
+		fees: {
+			ticker: "ARK",
```

Comment:
```suggestion
			ticker: "MARK",
```

as per task, or does vault determine the ticker elsewhere?